### PR TITLE
send resource link to LAWS

### PR DESCRIPTION
**Pull request by v-soukum**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.ADO/Framework/Abstracts/ADOSVTBase.ps1 [hunk 1]
```diff
@@ -4,6 +4,7 @@ class ADOSVTBase: SVTBase {
 	hidden [AzSKSettings] $AzSKSettings;
 	# below variable will be used by SVT's and overriden for each individual resource.
 	hidden [bool] $isResourceActive = $true;
+	hidden [string] $ResourceLink;
```
Comment (arvindsingh-cloud) [inline src/AzSK.ADO/Framework/Abstracts/ADOSVTBase.ps1:7]: purpose of adding ResourceLink in ADOSVTBase ?

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.AgentPool.ps1 [hunk 2]
```diff
@@ -31,6 +31,12 @@ class AgentPool: ADOSVTBase
             $this.isResourceActive = $false
         }
 
+        # getting resource link to send to LAWS.
+        if ($null -ne $svtResource.ResourceDetails.ResourceLink)
```
Comment (arvindsingh-cloud) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.AgentPool.ps1:35]: ??

File: src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Build.ps1 [hunk 3]
```diff
@@ -42,6 +42,12 @@ class Build: ADOSVTBase
             $this.isResourceActive = $false
         }
 
+        # getting resource link to send to LAWS.
```
Comment (arvindsingh-cloud) [inline src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Build.ps1:45]: ??

File: src/AzSK.ADO/Framework/Abstracts/ADOSVTBase.ps1 (hunk 1 above)
Comment (v-soukum) [reply src/AzSK.ADO/Framework/Abstracts/ADOSVTBase.ps1:7]: ADOSVTBase.ps1 file is inherited by all other SVT's thats why this gobal "ResourceLink" variable here.

**Review by Garima-msft — APPROVED** [review]

**Review by arvindsingh-cloud — APPROVED** [review]
looks good

**Review by abhaydaga — APPROVED** [review]

**Pull request merged**